src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,14 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    @Override
+    public List<Bulldozer> get() {
+        Bulldozer bulldozerFirst = new Bulldozer();
+        Bulldozer bulldozerSecond = new Bulldozer();
+        Bulldozer bulldozerThird = new Bulldozer();
+        return List.of(bulldozerFirst, bulldozerSecond, bulldozerSecond);

```

Comment:
```suggestion
        return List.of(new Bulldozer(), new Bulldozer(), new Bulldozer());

```
You don't need to create that local variables
Change it everywhere